lib/cubit/webhook/webhook_cubit.dart

```diff
@@ -77,8 +77,12 @@ class WebhookCubit extends Cubit<WebhookState> {
     if (lastUsedLnurlPay != null && lastUsedLnurlPay != webhookUrl) {
       await _invalidateLnurlPay(walletInfo, lastUsedLnurlPay);
     }
+    String? username = await _breezPreferences.getProfileName();
+    username = username?.replaceAll(' ', '');
```

Comment:
Do we need to worry about case sensitivity here? Does the LNURL service translate it to lowercase?

Reply:
It's lowercased by the LNURL service